system/jlib/jsmartsock.cpp

```diff
@@ -45,9 +45,22 @@ class SmartSocketListParser
         unsigned port = defport;
 
         char *saveptr;
-        char *ip = strtok_r(copyFullText, "|", &saveptr);
-        while (ip != NULL)
+        char *ip0 = strtok_r(copyFullText, "|", &saveptr);
```

Comment:
preexisting, but I like to encourage explicit/accurate/selfdocumenting var names...

Reply:
changed to hostportstr